packages/mobile/src/navigator/types.tsx

```diff
@@ -186,7 +186,7 @@ export type StackParamList = {
     isFromScan?: boolean
   }
   [Screens.VerificationEducationScreen]: { showSkipDialog: boolean } | undefined
-  [Screens.VerificationInputScreen]: undefined
+  [Screens.VerificationInputScreen]: { showHelpDialog: boolean } | undefined
```

Comment:
Why is this a nav param?

Reply:
This was needed to communicate between the NavBar Help button and the screen itself.
In retrospect I could have embedded the Dialog in the button 🤔 
Anyway I think we should probably change those dialogs to dedicated screens. This way they would not be unmounted abruptly when navigating away from the screen that hosted it.